secure_message_v2/logger_config.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import sys
+
+import structlog
+
+
+def logger_initial_config(log_level: str = "INFO") -> None:
+    def add_severity_level(_, method_name: str, event_dict: dict) -> dict:
+        """
+        Adds the log level to the event dict.
+        """
+        event_dict["severity"] = method_name
+        return event_dict
+
+    def add_service(_, __, event_dict: dict) -> dict:
+        """
+        Adds the service name (secure-message-v2) to the event dict.
+        """
+        event_dict["service"] = "secure-message-v2"
+        return event_dict
+
+    logging.basicConfig(format="%(message)s", stream=sys.stdout, level=logging.INFO)
+
+    structlog.configure(
+        processors=[
+            structlog.contextvars.merge_contextvars,
+            add_severity_level,  # type: ignore
+            add_service,  # type: ignore
```

Comment:
Ugly mypy issue, I am not going to worry about type hinting for an external config so ignored it